docs/config/environment.rst

```diff
@@ -128,9 +128,10 @@ Collector
 
 CrateDB uses the `G1`_ garbage collector by default.
 
-Before CrateDB 4.1 it defaulted to use the `Concurrent Mark Sweep` garbage
-collector. If you'd like to continue using CMS, you can switch setting the
-following :ref:`CRATE_JAVA_OPTS <conf-env-java-opts>`::
+Before CrateDB 4.1 it defaulted to use the `Concurrent Mark Sweep`_ garbage
+collector. Please note that CMS garbage collection is deprecated. If you'd
+like to continue using CMS, you can switch the following
+:ref:`CRATE_JAVA_OPTS <conf-env-java-opts>`: setting:
```

Comment:
I think we've to update this. CMS has been removed with JDK 14 https://openjdk.java.net/jeps/363